src/inference/src/dev/core_impl.cpp

```diff
@@ -812,10 +812,10 @@ void ov::CoreImpl::apply_auto_batching(const std::shared_ptr<const ov::Model>& m
     bool strictly_check_dims = true;
     if (deviceName.find("BATCH") != std::string::npos) {
         // explicitly enabled Auto-Batching
-        auto pos = deviceName.find_first_of(":");
+        auto pos = deviceName.find("BATCH:");
```

Comment:
could you please provide example why do we need it?

Reply:
For example: AUTO:BATCH:GPU

Reply:
Hi, @ilya-lavrenov  This modification to support 'AUTO:BATCH' with property: {"DEVICE_PROPERTIES", "{BATCH :{MULTI_DEVICE_PRIORITIES:**}}"}